Standards/scs-0112-v1-volume-type-standard.md

```diff
@@ -0,0 +1,220 @@
+
+### Glossary
+
+The following special terms are used throughout this standard document:
+| Term | Meaning |
+|---|---|
+| volume | OpenStack object, virtual drive which usually resides in a network storage backend |
+| AZ | Availability Zone |
+| Volume QoS | Quality of Service object for Volumes |
+
+## Motivation
+
+We want to standardize a few varieties of volume types. While a user can choose simple things like size when creating a volume, Volume Types define a few broader aspects of a volume. Encryption of volumes for example is solely decided by the volume type. While the option with which the volume will be replicated is a mix between definition in the volume type and backend specific configuration, but it's visibility can only be reached in the volume type. In the following part, we want to state, which varieties of volume types are REQUIRED, RECOMMENDED or only OPTIONAL within an SCS-compatible deployment.
+
+## Design Considerations
+
+All Considerations can be looked up in detail in the [Decision Record for the Volume Type Standard.](https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0111-v1-volume-type-decisions.md)
+
+To test whether a deployment has volume types with certain aspects, the discoverability of the parameters in the volume type has to be given. The following table shows, which aspects are considered in this standard and whether it is discoverable, and who can discover this. The comments state what can be done to help with the discoverability and other important restrictions.
+
+| Aspect | Standardize? | Discoverability | comments |
+| ---- | ---- | ---- | ------ |
+| encryption | **Recommended** | admin only | Upstream work for extra_spec: encrypted=True/False needed |
+| Backend name | - | - | - |
+| AZs | **Optional** | - | Upstream: setable extra_spec would help |
+| multiattach | **Optional** | yes | - |
+| Replication | **Recommended** | backend-dependend | Upstream: setable extra_spec would help |
+| Number of Replicas, etc | **Optional** | backend-dependend | Upstream: setable extra_spec would help |
+| Volume QoS | **Optional** | admin only | depends on the admin only volume qos object |
+
+In addition it is possible to use multiple of those aspects within one volume type. SCS will only ever look, if there is a volume type that has an aspect, but they don't have to be different volume types.
+Example: one volume type that uses LUKS-encryption with a ceph storage with inherent replication would fulfill all recommendations of this standard.
+
+## DEFAULT volume types
+
+There is always a default volume type defined in an OpenStack deployment. The SCS does not have any requirements about this volume type at this moment, instead deployers are free to choose what fits best in their environment.
```

Comment:
How is this "default" volume type called? This should be standardized in SCS clouds.

Reply:
I understand your point and we should add your explanations to the standard. IMO, the standard does not point out explicitly, that using no volume type, implies _default_ volume type is used, and the user can not expect any specific properties the default volume will have.

```suggestion
There is always a default volume type defined in an OpenStack deployment. The SCS does not have any requirements about this volume type at this moment, instead deployers are free to choose what fits best in their environment. Conversely, a cloud user can not expect any specific behavior or properties from default volume types.  
```